File: javascript/codeeval/099_calculate_distance.js
function pythagoras(line) {
    input = line.replace(/^\(/, '').replace(/\)$/, '').split(') (');
    coords_a = input[0].split(', ').map(function(x){ return parseInt(x) });
    coords_b = input[1].split(', ').map(function(x){ return parseInt(x) });
    side_a = Math.abs(coords_a[0] - coords_b[0]);
    side_b = Math.abs(coords_a[1] - coords_b[1]);
    console.log(Math.sqrt(Math.pow(side_a, 2) + Math.pow(side_b, 2)));
}

var fs  = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
        pythagoras(line)
    }
});

File: python/codeeval/099_calculate_distance.py
import sys
import re
from math import sqrt
def pyth(line):
    input = (re.sub('^\(', '',  re.sub('\)$', '', line))).split(') (')
    coords_a = [int(x) for x in input[0].split(', ')]
    coords_b = [int(x) for x in input[1].split(', ')]
    side_a = abs(coords_a[0] - coords_b[0])
    side_b = abs(coords_a[1] - coords_b[1])
    return int(sqrt((side_a * side_a) + (side_b * side_b)))

with open(sys.argv[1], 'r') as test_cases:
    for test in test_cases:
        print(pyth(test))
